sdcm/cluster_k8s/gke.py

```diff
@@ -34,7 +34,7 @@
 GKE_API_CALL_QUEUE_SIZE = 1000  # ops
 GKE_URLLIB_RETRY = 6  # How many times api request is retried before reporting failure
 
-SCYLLA_CLUSTER_CONFIG = sct_abs_path("sdcm/k8s_configs/cluster-gke.yaml")
+SCYLLA_CLUSTER_CONFIG = "/apps/k8s_configs/gke-cluster-chart-values.yaml"
```

Comment:
let's move `/apps/k8s_configs/` to a global var

Reply:
Done